data/upgrades/gunner.json

```diff
@@ -27,6 +27,14 @@
           "Gunner"
         ]
       }
+    ],
+    "restrictions": [
+      {
+        "factions": [
+          "Scum and Villainy"
+        ],
+        "character": "darthvader"
```

Comment:
I assume this is meant to use the XWS value? Unfortunately we cannot guarantee that all Vader cards have the same XWS value.
So I think this should use the full name `Darth Vader`.

Reply:
I wanted to keep it bound to the XWS value in order not to have to search all pilots/upgrades and just hit the XWS-value directly
But I see your point... :-/

Reply:
And alternative would would be to use an array of possible XWS values: `xws: ['darthvader', 'darthvader-crew'`.

Reply:
Would make sense also since it could be several different characters.
All other restrictions uses arrays so makes even more sense that way